From cf403ac6f86c8f1cfab45e310b4cc7babd0d775d Mon Sep 17 00:00:00 2001
From: Indrajit Das <indrajit-kumar.das@amd.com>
Date: Mon, 27 Feb 2017 12:57:07 +0530
Subject: [PATCH 1719/1722] drm/amd/dal: No audio output heard from DP panel

This is derived from the following commit: -
3b8d8825bc70c1adfbfb1cd715385aceea6f21ce on
https://cgit.freedesktop.org/~agd5f/linux/log/?h=amd-staging-4.9
---
 drivers/gpu/drm/amd/dal/dc/dce/dce_audio.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/dce/dce_audio.h b/drivers/gpu/drm/amd/dal/dc/dce/dce_audio.h
index 6c9fe17..0b62992 100644
--- a/drivers/gpu/drm/amd/dal/dc/dce/dce_audio.h
+++ b/drivers/gpu/drm/amd/dal/dc/dce/dce_audio.h
@@ -53,8 +53,8 @@
 		SF(DCCG_AUDIO_DTO_SOURCE, DCCG_AUDIO_DTO_SEL, mask_sh),\
 		SF(DCCG_AUDIO_DTO0_MODULE, DCCG_AUDIO_DTO0_MODULE, mask_sh),\
 		SF(DCCG_AUDIO_DTO0_PHASE, DCCG_AUDIO_DTO0_PHASE, mask_sh),\
-		SF(DCCG_AUDIO_DTO0_MODULE, DCCG_AUDIO_DTO0_MODULE, mask_sh),\
-		SF(DCCG_AUDIO_DTO0_PHASE, DCCG_AUDIO_DTO0_PHASE, mask_sh),\
+		SF(DCCG_AUDIO_DTO1_MODULE, DCCG_AUDIO_DTO1_MODULE, mask_sh),\
+		SF(DCCG_AUDIO_DTO1_PHASE, DCCG_AUDIO_DTO1_PHASE, mask_sh),\
 		SF(AZALIA_F0_CODEC_FUNCTION_PARAMETER_SUPPORTED_SIZE_RATES, AUDIO_RATE_CAPABILITIES, mask_sh),\
 		SF(AZALIA_F0_CODEC_FUNCTION_PARAMETER_POWER_STATES, CLKSTOP, mask_sh),\
 		SF(AZALIA_F0_CODEC_FUNCTION_PARAMETER_POWER_STATES, EPSS, mask_sh)
-- 
2.7.4

